manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -82,6 +82,9 @@ rules:
       - "signers"
     resourceNames:
     - "kubernetes.io/legacy-unknown"
+{{- range .Values.global.certSigners }}
+    - {{ . | quote }}
+{{- end }}
```

Comment:
I donot understand the whole bunch, IIUC k8s does not have `signers` resources at all

Reply:
So i doubt this is useless

Reply:
IC, the certmanager has a groups `"cert-manager.io"`, which has the `signers` resource, but quite different from `certificates.k8s.io` group.

Reply:
This bunch is not correct before and after

Reply:
It is part of the [kubenetes CSR API](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#signers/) 

And it is the same group in the [istio guide](https://istio.io/latest/docs/tasks/security/cert-management/custom-ca-k8s/#deploy-istio-with-default-cert-signer-info) that uses CSR API

Reply:
OK. never mind. CSR is special